src/styles/blocks/card-block/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  @for $starCount from 1 through 4 {
```

Comment:
The loop should run from 1 through 5, not 4. The task description specifies 4 cards per row, but it does not limit the number of stars to 4.